### ISSUE TITLE
Discussion: Do we want to introduce any form of QA (Quality Assurance) before an alpha release? 

### ISSUE THREAD

**Issue by pzaino** [RISC-OS-Community/DME-Core#7]
# Background

@RISC-OS-Community/code-reviews-team 
Given that DME is something we are creating for end-users of all level of experience on RISC OS, do we think it may be beneficial to introduce a "layer" of formal testing within the RISC OS Community before we release the code for a broader audience to test and report?

# Definition of Done

- [x] Idea proposed
- [ ] Idea being discussed
- [ ] We have reached a conclusion

**Comment by skymandr** [comment]
I think development and testing should all be in the open, so I would prefer a pre-release testing scheme in that case. Releases on Github can be tagged as pre-release, and such a build could be given to the community with a request for input in that context.

**Comment by pzaino** [comment]
Good point, pre-releases sound like a good way to ask the community for feedback and bug reports, until we believe it's "good enough" to tag an official release.
